scrapers/dc_onlyfans_fansdb/dc_onlyfans_fansdb.py

```diff
@@ -270,53 +258,68 @@ def find_gallery_path(gallery_id):
         return gallery["folder"]["path"]
 
     log.error(f'Path for gallery {gallery_id} could not be found')
-    print("{}")
+    print('null')
     sys.exit()
 
-def find_name_from_alias(alias):
+
+def get_performer_info(username, media_dir):
     """
-    Find and return a performer's name by their alias.
+    Resolve performer based on username
     """
-    perfs = stash.find_performers(
-        f={"aliases": {"value": alias, "modifier": "EQUALS"}},
+    req = stash.find_performers(
+        f={"aliases": {"value": username, "modifier": "EQUALS"}},
         filter={"page": 1, "per_page": 5},
-        fragment="name,id",
+        fragment="id, name",  # type: ignore
     )
-    log.debug(perfs)
-    if len(perfs) > 0:
-        perf = {"name": perfs[0]['name'], "stored_id": perfs[0]['id']}
-        return perf
-    return alias
+    log.debug(f'found performer(s): {req}')
+    res: Dict = {}
+    if len(req) == 1:
+        log.debug(f"Found performer id: {req[0]['id']}")
+        res['stored_id'] = req[0]['id']
+    res['name'] = username
+
+    images = get_performer_images(media_dir)
+    if images is not None:
+        res["images"] = images
 
-def find_parent_studio():
+    return [res]
+
+
+def get_studio_info(studio_name, studio_network):
     """
-    Find and return OnlyFans (network) `stored_id` else return only name
+    Resolve studio based on name and network
     """
-    studios = stash.find_studios(
-        f={"name": {"value": "OnlyFans (network)", "modifier": "EQUALS"}},
+    req = stash.find_studios(
+        f={"name": {"value": f"{studio_name} ({studio_network})", "modifier": "EQUALS"}},
```

Comment:
You could make the same change as I suggested for performers here if you'd like to support people having their studios aliased

You could even make the query more complex and find the studio with the `studio_name` as its name or alias AND the parent studio called `studio_network`, but that's probably overcomplicating things